[ADD] ecommerce/google merchant center: new page (#13957)
odoo/documentation: content/applications/websites/ecommerce/google_merchant_center.rst

   .. note::
      By enabling the :guilabel:`Google Merchant Center Data Source` option, your website will
      generate a dynamic `/gmc.xml` feed containing essential product information and availability.
      This feed can be :ref:`customized <ecommerce/GMC/localized-feed>` to include multiple
      languages and pricelists, ensuring your products are displayed correctly for different regions
      and audiences.

#. Go to the :abbr:`GMC (Google Merchant Center)` dashboard, navigate to the
   :menuselection:`Your business --> Products` tab in the left menu, and click :guilabel:`Add
   products`.
#. Choose :guilabel:`Add products from a file` and paste the URL of the copied file.

   .. important::
      Make sure to select all the countries where you intend to sell your products. You are not
      able to proceed without selecting at least one target country. Optionally, you have to choose
      a :guilabel:`feed label` as well.

      .. image:: google_merchant_center/gmc-select-countries.png
         :alt: Select countries in GMC.

#. Click :guilabel:`Continue`.

.. _ecommerce/GMC/localized-feed:

Localized feeds
===============

Languages/regions
-----------------

> It is helpful to create language-specific feeds for each country/language you sell in. You can
> always add new feeds by going to :guilabel:`Products` on the :abbr:`GMC` dashboard, clicking
> :guilabel:`Add products` and choosing :guilabel:`Add another product source` from the
> dropdown menu.

Reviewer: ```suggestion
It is helpful to create language-specific feeds for each country/language you sell in. To add a new feed, go to :guilabel:`Products` on the :abbr:`GMC` dashboard, click :guilabel:`Add products`, and select :guilabel:`Add another product source` from the dropdown menu.
```
I'd recommend using the imperative when we can

======================
Google Merchant Center
======================

Google Merchant Center is a specialized tool for ecommerce retailers that helps to manage product
data and make products visible to online shoppers across Google's platforms, including
Google Search, Google Maps, Google Shopping, and YouTube. It serves as a centralized hub where
retailers can upload and maintain details about their products such as images, prices, and
descriptions. By leveraging this tool, businesses can enhance their products' visibility as well as
enhance their advertising and sales performance.

Getting started
===============

To connect your ecommerce with the :abbr:`GMC (Google Merchant Center)` platform, proceed as
follows:

> #. First, go the `Google Merchant Center page <https://business.google.com/us/merchant-center/>`_.

Reviewer 2: ```suggestion
#. First, go to the `Google Merchant Center page <https://business.google.com/us/merchant-center/>`_.
```

#. Indicate that you sell products online, and enter :guilabel:`Your store's website`.

   .. image:: google_merchant_center/gmc-first-steps.png
      :alt: Your store's website screen

#. Click :guilabel:`Continue`, then click :guilabel:`Continue to Merchant Center`.

#. Enter your business details by adding the :guilabel:`Business name` and the
   :guilabel:`Registered country`, then click the :guilabel:`Continue to Merchant Center` button
   twice.

#. Go through the tasks to provide additional business information or click :guilabel:`Do it later`
   to access the dashboard directly.

Site ownership verification
---------------------------

In order to use the :abbr:`GMC (Google Merchant Center)` tool, verify your website's ownership
first. To do so, go to the :guilabel:`Business info` tab in the left menu, and click
:guilabel:`Confirm online store`.

There are several verification options:

- Via an ecommerce platform
- Via HTML tag or file
- Via :ref:`Google Tag Manager <analytics/google-tag-manager>`
- Via :ref:`Google Analytics <analytics/google-analytics>`

If you are not using Google Tag Manager or Google Analytics, the simplest method is adding

> the HTML tag.

Author: I will check and in case that's true, I will remove it

To verify using an HTML tag, follow the instructions:

.. image:: google_merchant_center/gmc-add-tag.png

>    :alt: Instructions to add the HTML tag.

Author: Hmm the image doesn't replace the text as it is is explained right after that but I can change the screenshot
